test/core/security/grpc_tls_certificate_provider_test.cc

```diff
@@ -536,13 +536,169 @@ TEST_F(GrpcTlsCertificateProviderTest, SuccessfulKeyCertMatch) {
   EXPECT_TRUE(*status);
 }
 
+TEST_F(GrpcTlsCertificateProviderTest, SuccessfulKeyMultipleCertMatch) {
+  absl::StatusOr<bool> status = PrivateKeyAndCertificateMatch(
+      private_key_2_, /*cert_chain*/ cert_chain_2_ + cert_chain_);
+  EXPECT_TRUE(status.ok());
```

Comment:
Suggest using ASSERT_TRUE here for ok status. Since that is a precondition for the below expectation. Similarly for other test cases.

Reply:
Noted, thanks.